modules/handleNavLinkClick.js

```diff
@@ -0,0 +1,17 @@
+const handleNavLinkClick = () => {
+  const navLinks = document.querySelectorAll('.nav-link');
+  navLinks.forEach((link) => {
+    link.addEventListener('click', (event) => {
+      const target = event.target.getAttribute('href');
+      const sections = document.querySelectorAll('section');
+      sections.forEach((section) => {
+        section.style.display = 'none';
+      });
+
+      document.querySelector(target).style.display = 'block';
+      window.location.href = target;
+    });
+  });
+};
+
+export default handleNavLinkClick;
```

Comment:
It is a requirement that when a link on the nav bar is clicked, the right section of the page is displayed😉. but currently, the links on your navigation are not working fine. kindly fix it😉.
<img width="1015" alt="image" src="https://github.com/Titus-Kiplagat/book-list-manager/assets/64643995/6536aba9-14da-4f60-9bc0-c4d02ffe1489">

You can take a look at[ this link ](https://www.loom.com/share/2a947b96e4d94c43b3f413aa065002d7?sid=6dc30b11-aafc-4655-a846-5b19d0a72051)for a piece of video evidence where I tested it.

Reply:
@Ibizugbe 
Firstly, thank you for reviewing my code. I would like to report that I've successfully implemented the navigation links functionality in the navbar. Now, when users click on the navigation links, they are correctly redirected to the corresponding page content. I've thoroughly tested this on my end and can confirm that it's working as intended.

To provide you with a visual demonstration, I've prepared a live demo showcasing the navigation links in action. You can access the demo through the following [link](https://www.loom.com/share/a6c8870ec5284306b537c79a221779cd?sid=52063fde-4fb0-4521-8bd3-a3144e65ef68).